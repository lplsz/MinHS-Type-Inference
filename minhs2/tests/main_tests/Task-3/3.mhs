main = letrec a :: Bool = b < c;
             b = 8;
             c :: Int = 7;
        in c;