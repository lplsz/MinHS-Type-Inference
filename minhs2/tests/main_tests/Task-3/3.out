main :: Int = letrec a :: Bool = (<) b c;
                     b :: Int = 8;
                     c :: Int = 7;
              in c;